setup.py

```diff
@@ -44,16 +44,16 @@
     install_requires=['django-model-utils>=4.0.0,<4.1.0'],
     extras_require={
         'qa': [
-            'black<=19.10b0',
+            'black<=20.8b1',
             'flake8<=3.9',
             'isort~=5.0',
-            'readme-renderer~=26.0',
+            'readme-renderer~=28.0',
```

Comment:
@devkapilbansal please can you double check this change doesn't affect the rstcheck negatively?

Reply:
No mentioned breaking changes in changelog, only python 3.9 support and a bug fixed is mentioned, check here: https://github.com/pypa/readme_renderer/blob/master/CHANGES.rst#280-2020-10-20

Reply:
There are no changes in the part of code that rstcheck uses